java/test/org/dartlang/vm/service/VmServiceTest.java

```diff
@@ -270,6 +270,7 @@ private static void startSample(File dartFile) {
     processArgs.add("--pause_isolates_on_start");
     processArgs.add("--observe");
     processArgs.add("--enable-vm-service=" + vmPort);
+    processArgs.add("--disable-service-auth-codes");
```

Comment:
We should really just handle the auth codes, this is just a test but still

Reply:
Agreed, but things are in flux right now. This flag exists in this dev build, won't exist in the next; instead they'll be a flag to opt in to the behavior. I'm going to wait until this settle down to address the auth token stuff.

Reply:
Fwiw, you could add the handling to just use the uri that it gives you via stdout, and that would continue to work regardless of how this option is set.

Reply:
I will lgtm since this is just a test and the risk is minimal, but it does make me sad that this flag exists at all, all the evidence I have seen so far is that everybody will just do the absolute minimal work unbreak their workflows (which is providing this flag instead of actually fixing the problem).

Reply:
We have updated the clients to use the full url (the dart and flutter IntelliJ plugins, the vs code plugin, and devtools). The impact of using the flags here is just scoped to the tests for this repo; I'm fine putting off the url parsing work here a bit.